--- tensorflow-1.9.0-rc2/tensorflow/workspace.bzl~	2018-07-02 22:07:51.000000000 +0000
+++ tensorflow-1.9.0-rc2/tensorflow/workspace.bzl	2018-07-03 17:18:05.642399039 +0000
@@ -685,10 +685,10 @@
       name = "cython",
-      sha256 = "6dcd30b5ceb887b2b965ee7ceb82ea3acb5f0642fe2206c7636b45acea4798e5",
+      sha256 = "f11ff41b4e47a42504de18fba47f60af5d72fb3be3073114f1651dad49653a2b",
       urls = [
-          "https://mirror.bazel.build/github.com/cython/cython/archive/3732784c45cfb040a5b0936951d196f83a12ea17.tar.gz",
-          "https://github.com/cython/cython/archive/3732784c45cfb040a5b0936951d196f83a12ea17.tar.gz",
+          "https://mirror.bazel.build/github.com/cython/cython/archive/7332cf1b83f1fa5af32749041abd39a6c1564e96.tar.gz",
+          "https://github.com/cython/cython/archive/7332cf1b83f1fa5af32749041abd39a6c1564e96.tar.gz",
       ],
-      strip_prefix = "cython-3732784c45cfb040a5b0936951d196f83a12ea17",
+      strip_prefix = "cython-7332cf1b83f1fa5af32749041abd39a6c1564e96",
       build_file = clean_dep("//third_party:cython.BUILD"),
       delete = ["BUILD.bazel"],
   )
--- tensorflow-1.9.0-rc2/tensorflow/c/eager/c_api.h~	2018-07-02 22:07:51.000000000 +0000
+++ tensorflow-1.9.0-rc2/tensorflow/c/eager/c_api.h	2018-07-06 19:34:09.121676588 +0000
@@ -76,7 +76,7 @@
 // Sets the default execution mode (sync/async). Note that this can be
 // overridden per thread using TFE_ContextSetAsyncForThread.
 TF_CAPI_EXPORT extern void TFE_ContextOptionsSetAsync(TFE_ContextOptions*,
-                                                      unsigned char async);
+                                                      unsigned char async_);
 
 TF_CAPI_EXPORT extern void TFE_ContextOptionsSetDevicePlacementPolicy(
     TFE_ContextOptions*, TFE_ContextDevicePlacementPolicy);
--- tensorflow-1.9.0-rc2/tensorflow/c/eager/c_api.h~	2018-07-06 19:34:09.000000000 +0000
+++ tensorflow-1.9.0-rc2/tensorflow/c/eager/c_api.h	2018-07-06 19:52:22.338661368 +0000
@@ -125,7 +125,7 @@
 
 // Overrides the execution mode (sync/async) for the current thread.
 TF_CAPI_EXPORT extern void TFE_ContextSetAsyncForThread(TFE_Context*,
-                                                        unsigned char async,
+                                                        unsigned char async_,
                                                         TF_Status* status);
 
 // Causes the calling thread to block till all ops dispatched in async mode
--- tensorflow-1.9.0-rc2/tensorflow/tools/pip_package/setup.py~	2018-07-02 22:07:51.000000000 +0000
+++ tensorflow-1.9.0-rc2/tensorflow/tools/pip_package/setup.py	2018-07-07 13:07:38.041796543 +0000
@@ -54,8 +54,8 @@
     'numpy >= 1.13.3',
     'six >= 1.10.0',
     'protobuf >= 3.6.0',
-    'setuptools <= 39.1.0',
-    'tensorboard >= 1.8.0, < 1.9.0',
+    'setuptools' ,
+    'tensorboard >= 1.8.0',
     'termcolor >= 1.1.0',
 ]
 
