--- tensorflow-1.11.0/tensorflow/tools/pip_package/setup.py~	2018-09-25 21:50:52.000000000 +0000
+++ tensorflow-1.11.0/tensorflow/tools/pip_package/setup.py	2018-09-28 18:38:37.534328613 +0000
@@ -56,7 +56,7 @@
     'numpy >= 1.13.3',
     'six >= 1.10.0',
     'protobuf >= 3.6.0',
-    'setuptools <= 39.1.0',
+    'setuptools',
     'tensorboard >= 1.11.0, < 1.12.0',
     'termcolor >= 1.1.0',
 ]
