--- tensorflow-1.9.0/tensorflow/workspace.bzl~	2018-08-06 01:19:40.000000000 +0000
+++ tensorflow-1.9.0/tensorflow/workspace.bzl	2018-08-07 16:31:06.442335711 +0000
@@ -107,11 +107,11 @@
   tf_http_archive(
       name = "eigen_archive",
       urls = [
-          "https://mirror.bazel.build/bitbucket.org/eigen/eigen/get/fd6845384b86.tar.gz",
-          "https://bitbucket.org/eigen/eigen/get/fd6845384b86.tar.gz",
+          "https://github.com/markdryan/eigen-git-mirror/archive/c6b265289d7360c0ed824da125a6afdb9476d55f.tar.gz",
+          "https://github.com/markdryan/eigen-git-mirror/archive/c6b265289d7360c0ed824da125a6afdb9476d55f.tar.gz",
       ],
-      sha256 = "d956415d784fa4e42b6a2a45c32556d6aec9d0a3d8ef48baee2522ab762556a9",
-      strip_prefix = "eigen-eigen-fd6845384b86",
+      sha256 = "db6b7b334f30266eb24a6975628de57c55d8c0e21c89022c2d30ccb26552ba42",
+      strip_prefix = "eigen-git-mirror-c6b265289d7360c0ed824da125a6afdb9476d55f",
       build_file = clean_dep("//third_party:eigen.BUILD"),
   )
 
